internal/install/recipe_installer.go

```diff
@@ -683,13 +685,34 @@ func (i *RecipeInstall) executeAndValidateWithProgress(ctx context.Context, m *t
 				i.progressIndicator.Canceled("Installing " + r.DisplayName)
 			} else {
 				i.progressIndicator.Fail("Installing " + r.DisplayName)
+				promptUserToSendErrorLogs(i)
```

Comment:
prompting the user to opt-in to sending errors to LogsAPI as well as temp file cleanup